homeassistant/components/aemet/sensor.py

```diff
@@ -130,4 +132,6 @@ def native_value(self):
         )
         if forecasts:
             forecast = forecasts[0].get(self.entity_description.key)
+            if self.entity_description.key == ATTR_FORECAST_TIME:
```

Comment:
Would be nice to move this case to a lambda function in the entity description.

Example:
https://github.com/home-assistant/core/blob/08a3140e6cbffc5677971f98191e24eea7085236/homeassistant/components/airly/sensor.py#L100

Reply:
Done :)